### PR TITLE
8.6.0-rc6

### PR DESCRIPTION
**Pull request by flomnes**

### CODE REVIEW

**Review by payetvin — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 Antares Changelog
 =================
 
+v8.6.0-rc6 (05/2023)
+--------------------
+## Format improvement
+ * Add bool for initial level optimization (#1312)
```
Comment (guilpier-code) [inline docs/CHANGELOG.md]: which levels are we talking about ?

**Review by guilpier-code — CHANGES_REQUESTED** [review]

File: docs/CHANGELOG.md (hunk 1 above)
Comment (flomnes) [reply docs/CHANGELOG.md]: #1312 as requested by @laurent-laporte-pro

Comment (guilpier-code) [reply docs/CHANGELOG.md]: Probably short term storage levels, but could we make it more clear ?

Comment (flomnes) [reply docs/CHANGELOG.md]: done

**Review by guilpier-code — APPROVED** [review]

**Pull request merged**